src/components/common/Chart/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,105 @@
+import { ResponsiveBar } from "@nivo/bar";
+import classNames from "classnames/bind";
+
+import utilFormatMoney from "@/utils/utilFormatMoney";
+
+import styles from "./BarChart.module.scss";
+import { ChartDataProps, mapChartDataToMonthlySummary, getMaxValue } from "./BarChart.utils";
+import { BarItem } from "./BarItem";
+
+const cx = classNames.bind(styles);
+
+interface CustomTooltipProps {
+  id: string,
+  value: number
+}
+
+const customTooltip = ({ id, value }: CustomTooltipProps) => {
+  let tooltipText = "";
+
+  switch (id) {
+    case "settlement":
+      tooltipText = "정산액";
+      break;
+    case "revenue":
+      tooltipText = "매출액";
+      break;
+    case "netIncome":
+      tooltipText = "회사 수익";
+      break;
+    default:
+  }
```

Comment:
```suggestion
  const tooltipTextMapper = {
    settlement: "정산액",
    revenue: "매출액",
    netIncome: "회사 수익",
  };

  const tooltipText = tooltipTextMapper[id];
```
요런 식으로 object mapping하면 어떨까요!

Reply:
수정완료!